data/json/items/tool_armor.json

```diff
@@ -1352,7 +1352,12 @@
         "flags" : ["VARSIZE", "WATER_FRIENDLY", "STURDY", "WAIST", "OVERSIZE"],
         "coverage" : 15,
         "material_thickness" : 2,
-        "use_action": [ "SHEATH_KNIFE", "HAMMER", "HACKSAW", "LUMBER", "REMOVE_ALL_MODS" ],
```

Comment:
> See commit notes - it's a temporary break as I don't think you can mix iuse_actor with iuse functions?

You can. You may need to declare the new action in item_actions.json (which you should do anyway).
The problem that prevented it was lack of the ability to get descriptions from `iuse`, which is now solved by providing them in item_actions.json.
I just tested it - it is possible to mix regular iuse and actors.

> the options would be to extend parsing of use_action or implement a custom menu for the survivor utility belt

Those would actually break some functionality, like using the belt from item action menu.